### DISCUSSION TITLE
Mythic points

### DISCUSSION THREAD

**Discussion by PerturaboS** [chromiecraft/chromiecraft#7617] [Suggestions]
Mythic points divided on 3 categories:
Gold - Rewards for each boss from BT or Hyjal,
Can be spend on items from both Raids. Each item cost same value, 28 - to make more friendly for new players and catch-up. 
Illidan/Archi items costs 42

Silver - T5 raids. Same situation. Can be gethered from SSC/TK. 
20 points for casual item, 30 for End Boss.

Copper - T4 Raids Mag/Grul/Kara (However, only Drakes from Kara drops points).
10 points regular item
15 points for Grull/Mag raid items + Aran trinket +40 Intel

Values of points can be changed, but I think 2 weeks for 1 item is OK if you raid whole bracket content.

What I would aslo suggest, is possibility to buy Lower currency:
You can buy 1 silver for 1 Gold, etc.

To open shops, for particular items, raids needs to be done on Mythic, like we have now.

Thanks,

**Comment by ZedNox** [comment]
Acculy agree like, but i would make every boss giving points, last one could maybe gives 2 and if you wanna make player not being able to unlock other raids stuff you just make what you have with BT now so requiring last boss kill for the possibility on what you can purchase by that raid

**Comment by krutoj87** [comment]
Are mythic points supposed to be bad luck protection or are they supposed to be a major way of gearing?

If they are supposed to be bad luck protection, you probably shouldn't accumulate enough points for an item after 2 weeks. Because going a few weeks without an item is completely normal. Bad luck only starts if you go without an upgrade for longer than normal. 
It might seem like a trivial thing, but it's a design decision, that might drasticly change how people play the game, so changes in those values shouldn't be made lightly.

**Reply by UndoUreche** [reply]
As I see it, the reward for a raid that doubles in duration is not merely bad luck protection. Rather, bad luck protection mechanics do not alter the gameplay and come as a bonus.

For example various warframe grinds rely on the player getting items randomly, but also getting a small amount of currency every time they attempt. Put enough currency together and you buy the item. There's no difficulty increase to preventing you from getting the currency.

**Reply by vvranceanu** [reply]
mythic points are meant to make active players be active for less time after releases. the point is to make everyone get their bis faster, allowing them to come to the realization they have no reason to stick to the frustrations of a less-than-perfect raid. those players can in turn then go on discord to cry over the low pop of the server

**Comment by PerturaboS** [comment]
Base for MytPoints, is to be bad luck solution, and yes, should be more than 2 weeks for Item. However on current state of server, end game TBC, IT should be working as gearing option.
If server one day gona be restarted, they can make it longer, but Mythics must be release in same time.

**Comment by Nyeriah** [comment]
Hello,

When mythics were reintroduced several concerns were raised and one of those concerns was the inflow of gear. Another one, was players being awarded gear for content they haven't completed, therefore:

- Shared points was something we abandoned long ago and is only present in the first raids (Mag, Gruul), which are both 1 boss raids
- Points were adjusted proportionally to the amount of bosses the raid contains, with the addition of now it being required to actually complete it in mythic before being any to purchase any items.

Mythic isn't, and wasn't, and shouldn't be the main source of gearing or something people should rely on exclusively to get the gear they want, which would eventually be the case if you could get a granted piece of gear too fast.

The mythic points awarded are a reward for completing the harder content, it isn't and was never meant to be an alternative way to play the game, - the main source the gear players want is still killing bosses, mythic ensures that after a while you *will* get the item you want, as opposed as before, as there were chances you could farm it for months and still not get the single piece of gear you want. 

Mythics are not a catch up mechanic as for that you already have catch up raids, designed for that purpose alone.

This suggestion, however, is acceptable, if:

- BT points are increased accordingly considering there are +5 bosses from Hyjal, so we would need to increase it by 1/5, at least
- Hyjal points are increased times 2,/2.5, considering there are +9 bosses in Black Temple to get points from
- Last boss/completion requirements remain in place

This is to address the concerns raised by the community:
- No unreasonable flow of gear or "purchasable gear"
- Players only be awarded for the content they acgtually complete

**Reply by PerturaboS** [reply]
BT Mythic takes aproximetly 2-2,5h. Hyjal at pre-nerf was around 1h.
That was my idea, that less time, less reward. But still Hyjal would be extra points to run for "most needed item".
After discussion in our guild, most people were interested in extra effort to get those +5 points weekly, instead  just farming BT.


- Players only be awarded for the content they acgtually 
Ok, but once you open Shop for SSC - means you did full clear at least one time. Posibilitie of buying t5 items, from t6 points gives players way of getting Tsunami/Sextant. 
Lot of  players, including me, run SSC basicly for whole T5 raids timing, and even during Hyjal release. The "unluck" of those items was insane, and considering, I attended every week raid, since December 23 till half of Hyjal released, I saw Sextant maybe 2 times. (Half of server might miss this item).
Allowing to sacrifice T6 points to get it, would be nice way see SSC 1-2 times, or just spend more expense points (Golden) for lower ItemLvl.

No unreasonable flow of gear or "purchasable gear"
About this point, BT Normal is kinda "free items" for players, as most of bosses are target dummy + low HP, comparing to Hyjal which was hard fight on every boss.
I will not mention release week of BT, which was half of what we have now.

**Reply by vvranceanu** [reply]
i think requiring players to engage in the gameplay loop and reward system of the game the willingly signed up to is a bit disrespectful to people's time. maybe it's a good idea to award people points if they go to bosses and emote /hello (maybe award more if they identify as being awarded more)

**Comment by UndoUreche** [comment]
I agree, however I want to add that the cost per item for every raid should be dependent on the number of available bosses, and be based on the idea that one has to be able to afford an item after 2 weeks of full clears.
I think this is more important than suggesting fixed prices.

**Reply by vvranceanu** [reply]
imo one has to be able to get what he wants when he wants. i think there should be a vendor with all the loot of the release for free, so people can get them as they will, why even wait the 2 weeks?

**Comment by vvranceanu** [comment]
This is a very well thought out and articulated idea, you can see the spark of intellect here.

If i may, i would like to contribute my personal emotional whims and experiences as well and present them as grand ideas. Please be aware i will be insulted if opposed, so positive vibes only.

So i left Warmane servers long ago because they were a baren bis raid-logging toxic wasteland(they since restarted some progression servers for some reason). When I came here, i was excited of the prospect of progression, but i realised the progression was too slow for my content-consuming-speed liking. So this gave me this genius idea(if i do say so myself): what if i demand that i can consume content even faster?! Why not use the rest of the time on Discord complaining the game is too easy and act like a unprovoked rabid dog instead?!

I therefore think this idea is in the right direction, but why stop here? I'd say make all raids give just points and points are useable to buy anything: gold, gems, honor, gear, whatever. Why require new players go to the hurdle of playing(and by new players i mean my alt, i really dont care about people trying to experience the game)? They should be there will all the rest, the raid-logging community. We need these people some of which log in for 4 hours a week to come trash the server on discord with us for low pop and bugs we don't even contribute to and we have no intention to(since playing 4 hours a week, really how much of a bother can it be to actually go investigate or report something). Even though i left Warmane, there was something about that end-of-content nothing to do feel that i really miss, so it would be really nice if we can experience that exact thing i left for here, BUT with every release.

I don't wanna get ahead of myself, but i will give you a preview of my next idea i will do a suggestion for soon: Warmane Blackrock PvP instant 70 bis-for-last-season free gear for EVERYTHING. We then can all be in Shattrath/Dalaran running in circles or mald on Discord, where all good servers finally go. Why wait?!

For those inclined to say there are other games, i should go play other game or do something else with my time if i stopped enjoying WoW and that i just now found this aimless experimental server where my dopamine dictated thoughts are upvoted by many like me who run on this hamster wheel for years on end: this server is not blizzlike and will never be.

Thank you,
